nix/mobile/android/build.nix

```diff
@@ -158,6 +161,7 @@ in stdenv.mkDerivation rec {
       --no-build-cache \
       --parallel \
       -Dmaven.repo.local='${deps.gradle}' \
+      -PreactNativeDevServerPort=${toString reactMetroPort} \
       assemble${gradleBuildType}
```

Comment:
Highlighting this part here:
* This `-PreactNativeDevServerPort` will configure the react-native build config to use the port override, otherwise it will default to `8081` and not connect to the metro server.

Reply:
If that's the case then what's the point of setting the `RCT_METRO_PORT` env variable?

Reply:
I think I was being to overly cautious with re-exporting the variable, but it doesn't seem we need it, so I've removed it.

Reply:
Cool.